flow/test/test_helper.sh

```diff
@@ -59,8 +60,13 @@ fi
 # Only enabled abort on error at the end to allow script to reach make issue
 set -e
 
-if [ ! -z ${MAKE_ISSUE+x} ]; then
-  $__make final_report_issue 2>&1 | tee -a $LOG_FILE
+if [ -n "${MAKE_ISSUE+x}" ]; then
+  $__make final_report_issue 2>&1 | tee -a "$LOG_FILE"
+fi
+
+if [ $ret -eq 0 ] && [ -f "$POWER_TCL_FILE" ]; then
```

Comment:
Here it would be preferable to check if the make file with the current design has a "simulate" and "power" target and let the design implement these targets however it wants.

Reply:
changed to separately checking whether make target exist